### PR TITLE
Dep upgrades

### PR DESCRIPTION
**Pull request by JakeSidSmith**
See individual commits

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,22 +35,22 @@
...
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-react": "^7.12.1",
-    "@babel/preset-typescript": "^7.12.1",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-const-enum": "0.0.5",
+    "@babel/preset-env": "^7.12.7",
+    "@babel/preset-react": "^7.12.7",
+    "@babel/preset-typescript": "^7.12.7",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-const-enum": "^1.0.1",
```
Comment (lCharlie123l) [inline package.json:45]: big version jump on this but I'm guessing they just actually went 1.0 rather than lots of breaking changes?

**Review by lCharlie123l — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (JakeSidSmith) [reply package.json:45]: No idea. 😂 it'll probably work - has no config, so nothing for us to update. I assume they've just fixed a bunch of bugs and are now happy with it.

Comment (JakeSidSmith) [reply package.json:45]: (they have no release notes or change log)

Comment (lCharlie123l) [reply package.json:45]: okey doke - that sounds a little worrying that they dont but also if we don't interact via a config then also it should still work. I guess you'll find out when you test it anyway :)

**Review by lCharlie123l — APPROVED** [review]

**Pull request merged**